### PR TITLE
Doco dev

### PR DESCRIPTION
**Pull request by bhamail**
Captures the steps I followed to setup a dev environment.

Old steps are kept because the appear to work also. My environment had a issue where the `~/.netrc` file needed stricter permissions (644 to 600). Fixed by: `chmod 600 ~/.netrc`

For reference, here is a snippet the errors I was seeing:
```
...
Failures:

  1) Chelsea::Gems when talking to OSS Index given a valid Gemfile.lock can collect dependencies, query, and print results
     Failure/Error: expect { command.execute }.to_not raise_error
     
       expected no Exception, got #<Netrc::Error: Permission bits for '/Users/bhamail/.netrc' should be 0600, but are 644> with backtrace:
         # ./lib/chelsea/oss_index.rb:54:in `call_oss_index'
         # ./lib/chelsea/oss_index.rb:45:in `block in get_vulns'
         # ./lib/chelsea/oss_index.rb:44:in `each'
         # ./lib/chelsea/oss_index.rb:44:in `get_vulns'
         # ./lib/chelsea/gems.rb:101:in `audit'
         # ./lib/chelsea/gems.rb:56:in `execute'
         # ./spec/unit/gems_spec.rb:30:in `block (5 levels) in <top (required)>'
         # ./spec/unit/gems_spec.rb:30:in `block (4 levels) in <top (required)>'
     # ./spec/unit/gems_spec.rb:30:in `block (4 levels) in <top (required)>'

  2) Chelsea::IQClient with defaults with an generated dependencies sbom should be able to submit an sbom
...
```

cc @bhamail / @DarthHater / @brittanybelle / @gmohre

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,7 +174,40 @@ Report URL: http://localhost:8070/ui/links/application/testapp/report/0e0f469269
...
+We suggest using [rbenv](https://github.com/rbenv/rbenv) to setup a reliable ruby development environment.
+
+Follow the [installation steps](https://github.com/rbenv/rbenv#installation). 
+For macos (10.15.7), there was a problem with step 2, with: `$ rbenv init`. The command 
+printed suggested editing `~/.bashrc`; however, this did not work in our case (even after an OS reboot),
+and we had to instead edit `~/bash_profile`. To sanity check your installation, you should see the 
+`.rbenv` directory early in your PATH, e.g.:
+```
+$ echo $PATH
+/Users/<username>/.rbenv/shims:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/usr/local/go/bin:/opt/X11/bin:/Applications/Wireshark.app/Contents/MacOS:/usr/local/git/bin
```
Comment (DarthHater) [inline README.md]: Maybe snip a bit out of here (like LaTex, Wireshark, etc...)

File: README.md [hunk 2]
```diff
@@ -174,7 +174,40 @@ Report URL: http://localhost:8070/ui/links/application/testapp/report/0e0f469269
...
+```
+bundle install
+```
+
+Run tests:
+```
+bundle exec rspec
+```
+
+~~Old Setup Steps~~
```
Comment (DarthHater) [inline README.md]: Probably just nix this and kill the `setup` file, no reason to keep it around!

File: README.md (hunk 1 above)
Comment (bhamail) [reply README.md]: Done 3571f20

File: README.md (hunk 2 above)
Comment (bhamail) [reply README.md]: Roger bcc3ddf

**Pull request merged**